codec_specs.md

```diff
@@ -764,3 +784,70 @@ Codec Name: VobBtn Buttons
 
 Description: Based on [MPEG/VOB PCI packets](http://dvd.sourceforge.net/dvdinfo/pci_pkt.html). The file contains a header consisting of the string "butonDVD" followed by the width and height in pixels (16 bits integer each) and 4 reserved bytes. The rest is full [PCI packets](http://dvd.sourceforge.net/dvdinfo/pci_pkt.html).
 
+## Block Addition Mappings
+
+Registered `BlockAddIDType` are:
+
+### Use BlockAddIDValue
+
+Block type identifier: 0
+
+Block type name: Use BlockAddIDValue
+
+Description: This value indicates that the actual type is stored in `BlockAddIDValue` instead. This value is expected to be used when it is important to have a strong compatibility with players or derived formats not supporting `BlockAdditionMapping` but using `BlockAdditions` with an unknown `BlockAddIDValue`, and SHOULD NOT be used if it is possible to use another value.
+
+### Opaque data
+
+Block type identifier: 1
+
+Block type name: Opaque data
+
+Description: the `BlockAdditional` data is interpreted as opaque additional data passed to the codec with the Block data. `BlockAddIDValue` MUST be 1.
+
+### ITU T.35 metadata
+
+Block type identifier: 4
+
+Block type name: ITU T.35 metadata
+
+Description: the `BlockAdditional` data is interpreted as ITU T.35 metadata as defined by ITU-T T.35 terminal codes. `BlockAddIDValue` MUST be 4.
+
+### avcE
+
+Block type identifier: 0x61766345
+
+Block type name: Dolby Vision enhancement-layer AVC configuration
+
+Type Description: the `BlockAddIDExtraData` data is interpreted as `AVCDecoderConfigurationRecord` structure for the enhancement layer as defined in [@!ISO.14496-15.2014], as described in [Dolby Vision Streams File Format](ttps://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-bitstreams-within-the-iso-base-media-file-format-v2.1.2.pdf). This extension MUST NOT be used if `Codec ID` is not `V_MPEG4/ISO/AVC`.
+
+### dvcC
+
+Block type identifier: 0x64766343
+
+Block type name: Dolby Vision configuration
+
+Description: the `BlockAddIDExtraData` data is interpreted as `DOVIDecoderConfigurationRecord` structure as defined in [Dolby Vision Streams File Format](https://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-bitstreams-within-the-iso-base-media-file-format-v2.1.2.pdf), for Dolby Vision profiles less than and equal to 7.
```

Comment:
The link should be turned into a reference, otherwise the TXT version of the RFC will likely miss the link.